packages/shared/src/components/modals/award/BuyCoresModal.tsx

```diff
@@ -323,7 +323,7 @@ const BuyCoreDesktop = () => {
 
 const BuyFlow = ({ ...props }: ModalProps): ReactElement => {
   const { setActiveModal } = useGiveAwardModalContext();
-  const isMobile = useViewSizeClient(ViewSize.MobileL);
+  const isMobile = useViewSize(ViewSize.MobileL);
```

Comment:
useViewSizeClient can return undefined to avoid SSR issue so it requires undefined to also be handled, but this is fine since its modal 👍

Reply:
Yeah. And modal is only client side, so no need to worry about SSR there :D